source/frontend/src/features/mapSideBar/acquisition/models/PayeeOptionModel.ts

```diff
@@ -70,18 +70,20 @@ export class PayeeOption {
 
   public static fromApi(compReqPayee: ApiGen_Concepts_CompReqPayee): PayeeOption {
     let payee: PayeeOption = null;
-    if (compReqPayee.acquisitionOwnerId) {
+    if (isValidId(compReqPayee.acquisitionOwnerId)) {
       payee = PayeeOption.createOwner(compReqPayee.acquisitionOwner, compReqPayee);
-    } else if (compReqPayee.acquisitionFileTeamId) {
+    } else if (isValidId(compReqPayee.acquisitionFileTeamId)) {
       payee = PayeeOption.createTeamMember(compReqPayee.acquisitionFileTeam, compReqPayee);
-    } else if (compReqPayee.interestHolderId) {
+    } else if (isValidId(compReqPayee.interestHolderId)) {
       payee = PayeeOption.createInterestHolder(compReqPayee.interestHolder, compReqPayee);
     }
-    payee.rowVersion = compReqPayee.rowVersion;
```

Comment:
this was one of the sources of NPE - when all payees are null then this variable is null so it cannot take an assignment of "rowVersion"